pkg/client/cli/cliutil/daemon.go

```diff
@@ -0,0 +1,186 @@
+package cliutil
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	// nolint:depguard // Because we won't ever .Wait() for the process and we'd turn off
+	// logging, using dexec would just be extra overhead.
+	"os/exec"
+
+	"github.com/kballard/go-shellquote"
```

Comment:
We don't really need this extra dep. We have `logging.ShellString(exe, args)` instead of `shellquote.Join(args...)`. I'm changing that back in my Windows PR so you don't need to change it here.